simple/README.md

```diff
@@ -13,6 +13,8 @@ python3 -m stats.main
 By default it reads inputs CSVs from the `.data/input` folder and outputs
 a sqlite `datacommons.db` file in the `.data/output` folder.
 
+a DC_API_KEY environment variable can be passed in, to enable API lookups.
```

Comment:
Consider rewording to:

To enable Data Commons API lookups, set a `DC_API_KEY` environment variable. Get the API key by sending an email to support@datacommons.org.